From f3bbc0ba0e34ea1b34b106228c02d85865ed904a Mon Sep 17 00:00:00 2001
From: Nate Graham <nate@kde.org>
Date: Tue, 24 Mar 2020 08:15:12 -0600
Subject: [PATCH] Stop multiplying duration values

Summary:
After D28144, some animations are now too long because they were inappropriately
multiplying a duration value. That's no longer necessary, so let's remove the
multiplication.

Depends on D28144

Test Plan: Various things that were a bit too slow before feel just right now

Reviewers: #vdg, #plasma

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D28239
---
 applets/taskmanager/package/contents/ui/Task.qml            | 2 +-
 applets/taskmanager/package/contents/ui/TaskList.qml        | 2 +-
 .../contents/configuration/PanelConfiguration.qml           | 4 ++--
 desktoppackage/contents/explorer/WidgetExplorer.qml         | 6 +++---
 toolboxes/desktoptoolbox/contents/ui/ToolBoxButton.qml      | 6 +++---
 5 files changed, 10 insertions(+), 10 deletions(-)

--- applets/taskmanager/package/contents/ui/Task.qml.orig	2020-05-05 14:51:18 UTC
+++ applets/taskmanager/package/contents/ui/Task.qml
@@ -293,7 +293,7 @@ MouseArea {
         Timer {
             id: timer
 
-            interval: units.longDuration * 2
+            interval: units.longDuration
             repeat: false
 
             onTriggered: {
--- applets/taskmanager/package/contents/ui/TaskList.qml.orig	2020-05-05 14:51:18 UTC
+++ applets/taskmanager/package/contents/ui/TaskList.qml
@@ -34,7 +34,7 @@ Flow {
             NumberAnimation {
                 properties: "x, y"
                 easing.type: Easing.OutQuad
-                duration: units.longDuration * 2
+                duration: units.longDuration
             }
 
             PropertyAction { target: taskList; property: "animating"; value: false }
--- desktoppackage/contents/configuration/PanelConfiguration.qml.orig	2020-05-05 14:51:18 UTC
+++ desktoppackage/contents/configuration/PanelConfiguration.qml
@@ -103,7 +103,7 @@ PlasmaCore.FrameSvgItem {
                     return panel.screenGeometry.y + panel.screenGeometry.height - panel.height - panel.distance
                 }
             }
-            duration: units.shortDuration * 3
+            duration: units.shortDuration
         }
 
         NumberAnimation {
@@ -122,7 +122,7 @@ PlasmaCore.FrameSvgItem {
                     return panel.screenGeometry.y + panel.screenGeometry.height - panel.height - configDialog.height - panel.distance
                 }
             }
-            duration: units.shortDuration * 3
+            duration: units.shortDuration
         }
     }
 //END Animations
--- desktoppackage/contents/explorer/WidgetExplorer.qml.orig	2020-05-05 14:51:18 UTC
+++ desktoppackage/contents/explorer/WidgetExplorer.qml
@@ -306,7 +306,7 @@ Item {
                     properties: "x"
                     from: -list.width
                     to: 0
-                    duration: units.shortDuration * 3
+                    duration: units.shortDuration
 
                 }
             }
@@ -316,7 +316,7 @@ Item {
                 NumberAnimation {
                     properties: "x"
                     to: list.width
-                    duration: units.shortDuration * 3
+                    duration: units.shortDuration
                 }
             }
 
@@ -329,7 +329,7 @@ Item {
             displaced: Transition {
                 NumberAnimation {
                     properties: "y"
-                    duration: units.shortDuration * 3
+                    duration: units.shortDuration
                 }
             }
         }
--- toolboxes/desktoptoolbox/contents/ui/ToolBoxButton.qml.orig	2020-05-05 14:51:18 UTC
+++ toolboxes/desktoptoolbox/contents/ui/ToolBoxButton.qml
@@ -54,14 +54,14 @@ Item {
     }
     Behavior on rotation {
         NumberAnimation {
-            duration: units.shortDuration * 3;
+            duration: units.shortDuration;
             easing.type: Easing.InOutExpo;
         }
         enabled: visible
     }
     Behavior on x {
         NumberAnimation {
-            duration: units.shortDuration * 3;
+            duration: units.shortDuration;
             easing.type: Easing.InOutExpo;
         }
         enabled: visible
@@ -69,7 +69,7 @@ Item {
     }
     Behavior on y {
         NumberAnimation {
-            duration: units.shortDuration * 3;
+            duration: units.shortDuration;
             easing.type: Easing.InOutExpo;
         }
         enabled: visible
